CODE_OF_CONDUCT.md

```diff
@@ -30,7 +30,9 @@ Headcoders may close your PR at their discretion if your PR history has little f
 
 All Pull Requests are expected to be tested prior to submission. If a submitted Pull Request fails to pass CI checks, the likelihood of it being merged will be significantly lower. If you can't take the time to compile/test your Pull Request, do not expect a warm reception.
 
-It is expected that contributors discuss larger changes on the forums or repository discussion tab prior to coding a Pull Request. The amount of time spent on any given Pull Request is not relevant. Repo staff are not responsible for contributors wasting their time creating features nobody asked for.
+It is expected that contributors discuss larger changes on the forums or repository discussion tab prior to coding a Pull Request. The amount of time spent on any given Pull Request is not relevant. Repo staff are not responsible for contributors wasting their time creating features nobody asked for. Be sure to inform the corresponding teams about the forum post or discussion.
```

Comment:
```suggestion
It is expected that contributors discuss larger changes on the forums or repository discussion tab prior to opening a Pull Request. The amount of time spent on any given Pull Request is not relevant. Repo staff are not responsible for contributors wasting their time creating features nobody asked for. Be sure to inform the corresponding teams about the forum post or discussion.
```

Reply:
IIRC the original meaning of that sentence was indeed before opening. To avoid people wasting their time

Reply:
suggestion updated